mirgecom/initializers.py

```diff
@@ -1001,36 +976,46 @@ def __call__(self, x_vec, cv, eos=None, **kwargs):
         if x_vec.shape != (self._dim,):
             raise ValueError(f"Expected {self._dim}-dimensional inputs.")
 
-        gamma = eos.gamma()
+        if not isinstance(eos, IdealSingleGas):
+            temperature = eos.temperature(cv, tseed)
+            gamma = eos.gamma(cv=cv, temperature=temperature)
+        else:
+            temperature = eos.temperature(cv)
+            gamma = eos.gamma(cv=cv)
```

Comment:
I'm pretty sure this can just be:
```suggestion
        temperature = eos.temperature(cv, tseed)
        gamma = eos.gamma(cv, temperature)
```

Reply:
Done.